public/lib/downloadHttpConcurrentProgress.js

```diff
@@ -77,11 +79,12 @@
      * @return error object
      */
     downloadHttpConcurrentProgress.prototype.onTestError = function (result) {
-        if (this._running) {
-            this.clientCallbackError(result);
-            clearInterval(this.interval);
-            this._running = false;
-        }
+      console.dir(result);
```

Comment:
a quick question: will it be any help to us having the console here.. without have it in the splunk?